wordpress-mobile/WordPress-Android
# Extract and refactor EditPostActivity.onEditorEvent

Fixes #10793 

To test:
1. Open an editor
2. Type something
3. Select eg. Bold
4. Make sure the event gets tracked - check LogCat

PR submission checklist:
